src/routes/register_request.rs

```diff
@@ -139,3 +142,46 @@ pub(super) async fn handle_image(
     .await
     .unwrap()
 }
+
+pub(super) async fn handle_font(
```

Comment:
axum repo has an example how to do this https://github.com/tokio-rs/axum/blob/main/examples/multipart-form/src/main.rs